ui/apps/platform/src/Containers/AccessControl/PermissionSets/ResourceDescription.tsx

```diff
@@ -17,7 +20,8 @@ const resourceDescriptions: Record<ResourceName, string> = {
         'Read: View backup integrations and configurations. Write: Modify backup integrations and configurations.',
     CVE: 'Internal use only',
     Cluster: 'Read: View secured clusters. Write: Add, modify, or delete secured clusters.',
-    Compliance: 'Read: View compliance standards and results. Write: N/A',
+    Compliance:
+        'Read: View compliance standards, results and runs. Write: Add, modify or delete scheduled compliance runs',
```

Comment:
Add serial comma please: `..., and runs...` and `... modify, or delete...`. Also period at the end of the sentence.

Reply:
Addressed in follow-up PR.